test/experimental/jumpdest_analysis.hpp

```diff
@@ -0,0 +1,26 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2020 The evmone Authors.
```

Comment:
Also exists from 2020? I have seen only reference implementation before.

Reply:
> Also exists since 2020? I have seen only reference implementation before.

Reply:
Yes, this is this old.